docs/security/seccomp.md

```diff
@@ -0,0 +1,66 @@
+<!-- [metadata]>
++++
+title = "Seccomp security profiles for Docker"
+description = "Enabling seccomp in Docker"
+keywords = ["seccomp, security, docker,  documentation"]
+[menu.main]
+draft = true
```

Comment:
@jfrazelle Is this meant to be a draft? Draft's don't go out into the live docs.

When would you like it to go live?